src/docs/pages/BreadcrumbPage.tsx

```diff
@@ -1,30 +1,33 @@
 import { FC } from 'react';
 import { HiHome } from 'react-icons/hi';
+import Breadcrumb from '../../lib/components/Breadcrumb';
 
-import { Breadcrumb } from '../../lib';
 import { CodeExample, DemoPage } from './DemoPage';
 
 const BreadcrumbPage: FC = () => {
   const examples: CodeExample[] = [
     {
       title: 'Default breadcrumb',
       code: (
-        <Breadcrumb
-          items={[
-            {
-              icon: HiHome,
-              label: 'Home',
-              href: '/breadcrumb',
-            },
-            {
-              label: 'Projects',
-              href: '/breadcrumb',
-            },
-            {
-              label: 'Flowbite React',
-            },
-          ]}
-        />
+        <Breadcrumb>
+          <Breadcrumb.Item href="#" icon={HiHome}>
+            Home
+          </Breadcrumb.Item>
+          <Breadcrumb.Item href="#">Projects</Breadcrumb.Item>
+          <Breadcrumb.Item icon={HiHome}>Flowbite React</Breadcrumb.Item>
+        </Breadcrumb>
+      ),
+    },
+    {
+      title: 'Solid background',
+      code: (
+        <Breadcrumb className="bg-gray-50 py-3 px-5 dark:bg-gray-900">
+          <Breadcrumb.Item href="#" icon={HiHome}>
+            Home
+          </Breadcrumb.Item>
+          <Breadcrumb.Item href="#">Projects</Breadcrumb.Item>
+          <Breadcrumb.Item icon={HiHome}>Flowbite React</Breadcrumb.Item>
```

Comment:
I think in all examples the final element does not have an icon in the official documentation